src/components/CreateCourse/CreateCourse.jsx

```diff
@@ -0,0 +1,226 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import PropTypes from 'prop-types';
+
+import { validateNewCourseField } from '../../helpers/newCourseValidation';
+import { formatNewCourseDuration } from '../../helpers/formatNewCourseDuration';
+import { formatNewCourseDate } from '../../helpers/formatNewCourseDate';
+
+import AuthorItem from './components/AuthorItem';
+import Button from '../../common/Button/Button';
+import Input from '../../common/Input/Input';
+import Textarea from '../../common/Textarea/Textarea';
+
+import styles from './CreateCourse.module.css';
+
+const CreateCourse = ({ courses, setCourses }) => {
+	const [titleValue, setTitleValue] = useState('');
+	const [descriptionValue, setDescriptionValue] = useState('');
+	const [durationValue, setDurationValue] = useState('');
+	const [authorNameValue, setAuthorNameValue] = useState('');
+	const [authors, setAuthors] = useState([]);
+	const [courseAuthors, setCourseAuthors] = useState([]);
+	const [titleValid, isTitleValid] = useState(true);
+	const [descriptionValid, isDescriptionValid] = useState(true);
+	const [durationValid, isDurationValid] = useState(true);
+	const [authorNameValid, isAuthorNameValid] = useState(true);
+
+	const navigateCourses = useNavigate();
+
+	const handleTitleChange = (value) => {
+		setTitleValue(value);
+		isTitleValid(validateNewCourseField(value));
+	};
+
+	const handleDescriptionChange = (value) => {
+		setDescriptionValue(value);
+		isDescriptionValid(validateNewCourseField(value));
+	};
+
+	const handleDurationChange = (value) => {
+		setDurationValue(value);
+		isDurationValid(validateNewCourseField(value));
```

Comment:
```suggestion
		isDurationValid(value > 0);
```

still using this method for validation, should not see this error, 5 minute is a valid length

![2023-08-22_16h51_04](https://github.com/jurica-zecevic/react/assets/30794349/2581c8a0-9849-49d0-977a-8b1ad358d575)